src/components/CommentList.jsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from "react";
+import { fetchCommentsForReviewId } from "../api";
+import CommentCard from "./CommentCard";
+
+
+const CommentList = ({ reviewId }) => {
+    const [comments, setComments] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(() => {
+      if (reviewId) {
+        fetchCommentsForReviewId(reviewId)
+          .then((commentsData) => {
+            console.log('Comments data received:', commentsData);
+            if (commentsData instanceof Array) {
+              setComments(commentsData)
+            }
+            setIsLoading(false)
+          })
+
+          .catch((error) => {
```

Comment:
Same as the api, there is no need for a catch block here as it will not be achieving anything.